.gitignore

```diff
@@ -1,10 +1,7 @@
 bin/
 target/
-metadata-gen/
```

Comment:
The metadata-gen doesn't exist anymore since 8.x, when that was refactored to load original Checkstyle metadata.